clients/drcachesim/common/trace_entry.h

```diff
@@ -431,6 +431,12 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_RSEQ_ENTRY,
 
+    /**
+     * Indicates the PC of the abort handler for the following rseq region.  Always
+     * follows #TRACE_MARKER_TYPE_RSEQ_ENTRY.
+     */
+    TRACE_MARKER_TYPE_RSEQ_HANDLER,
```

Comment:
> There is no way to obtain the precise interruption PC from the kernel

So for all types of rseq abort (signal, thread migration etc), the pc shown by the kernel and DR is the abort handler pc.

I'm thinking why we need to specify the handler address separately when TRACE_MARKER_TYPE_RSEQ_ABORT already has it. Maybe the invariant checker should just make sure that all instances of TRACE_MARKER_TYPE_RSEQ_ABORT have the same address.

Reply:
Hmm, I did not consider this: I think you are right and we could eliminate this new marker.  Let me try to adjust the code for that.

Reply:
Confirmed: the new marker can be eliminated.  Done.